### PR TITLE
Add ClientResultSample

### PR DESCRIPTION
**Pull request by JialinXin**
image

### CODE REVIEW

File: samples/RazorComponentSample/App.razor [hunk 1]
```diff
@@ -1,5 +0,0 @@
-﻿@*
```
Comment (vicancy) [inline samples/RazorComponentSample/App.razor:1]: Are these files removed intentionally?

File: samples/ClientResultSample/wwwroot/lib/bootstrap/dist/js/bootstrap.min.js.map [hunk 2]
```diff
@@ -0,0 +1 @@
...\nenableDismissTrigger(Toast)\n\n/**\n * ------------------------------------------------------------------------\n * jQuery\n * ------------------------------------------------------------------------\n * add .Toast to jQuery only if jQuery is present\n */\n\ndefineJQueryPlugin(Toast)\n\nexport default Toast\n","/**\n * --------------------------------------------------------------------------\n * Bootstrap (v5.1.0): index.umd.js\n * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)\n * --------------------------------------------------------------------------\n */\n\nimport Alert from './src/alert'\nimport Button from './src/button'\nimport Carousel from './src/carousel'\nimport Collapse from './src/collapse'\nimport Dropdown from './src/dropdown'\nimport Modal from './src/modal'\nimport Offcanvas from './src/offcanvas'\nimport Popover from './src/popover'\nimport ScrollSpy from './src/scrollspy'\nimport Tab from './src/tab'\nimport Toast from './src/toast'\nimport Tooltip from './src/tooltip'\n\nexport default {\n  Alert,\n  Button,\n  Carousel,\n  Collapse,\n  Dropdown,\n  Modal,\n  Offcanvas,\n  Popover,\n  ScrollSpy,\n  Tab,\n  Toast,\n  Tooltip\n}\n"]}
```
Comment (vicancy) [inline samples/ClientResultSample/wwwroot/lib/bootstrap/dist/js/bootstrap.min.js.map:1]: remove not used files

File: samples/RazorComponentSample/App.razor (hunk 1 above)
Comment (JialinXin) [reply samples/RazorComponentSample/App.razor:1]: Yes. It's some middle version of Blazor sample and somehow not removed clean...

File: samples/ClientResultSample/ClientResultSample.csproj [hunk 3]
```diff
@@ -1,9 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
```
[the remark this answers is not in the document]
Comment (JialinXin) [reply samples/ClientResultSample/ClientResultSample.csproj:4]: No. Add and remove. Somehow it detects as update...

**Comment by xingsy97** [comment]
How about adding error completion & error handling like [this](https://github.com/xingsy97/azure-signalr/commit/d33a63fe33db2ea6a25b57f810159d31b3090bf2) ?

**Comment by JialinXin** [comment]
> How about adding error completion & error handling like [this](https://github.com/xingsy97/azure-signalr/commit/d33a63fe33db2ea6a25b57f810159d31b3090bf2) ?

`try catch` looks good to me. But `SendErrorButton` from client looks confused in current scenario. Let's figure out suitable ones in other samples.

File: samples/ChatSample/ChatSample.Net70/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+## Example app that makes use of SignalR Client Results
+
+Requires NET7.0 Preview7 or later SDK/Runtime.
+Can be installed from https://dotnet.microsoft.com/en-us/download/dotnet/7.0.
+
+### Usage
+
+F5 (ctrl-F5) to launch the server and get the URL for the app endpoint.
+
+#### Using client results in a Hub method
```
Comment (vicancy) [inline samples/ChatSample/ChatSample.Net70/README.md:10]: also add how to set the Azure SignalR connection string?

**Review by vicancy — APPROVED** [review]

**Pull request merged**